app/src/main/java/org/cru/godtools/activity/GlobalDashboardActivity.kt

```diff
@@ -0,0 +1,38 @@
+package org.cru.godtools.activity
+
+import android.app.Activity
+import android.content.Intent
+import android.os.Bundle
+import org.cru.godtools.R
+import org.cru.godtools.fragment.GlobalDashboardFragment
+
+private const val TAG_GLOBAL_DASHBOARD_FRAGMENT = "globalDashboardFragment"
+
+fun Activity.startGlobalDashboardActvity() {
+    Intent(this, GlobalDashboardActivity::class.java)
+        .also { startActivity(it) }
```

Comment:
let's simplify this to: `startActivity(Intent(this, GlobalDashboardActivity::class.java))`